jaclang_jaseci/core/architype.py

```diff
@@ -0,0 +1,539 @@
+"""Core constructs for Jac Language."""
```

Comment:
Does this file need to be updated? I think most of it will be in jaclang after the PR gets merged right?